kornia/feature/__init__.py

```diff
@@ -4,7 +4,7 @@
                         harris_response,
                         gftt_response,
                         hessian_response)
-from .nms import NonMaximaSuppression2d, non_maxima_suppression2d
+from .nms import NonMaximaSuppression2d, nms2d, NonMaximaSuppression3d, nms3d
```

Comment:
i wouldn't do that, add alias if you want but this will break backward compatibility